.github/renovate-aws-marketplace.json5

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
@jessesimpson36 I believe that file is copied from the Helm repo.
Could we please clean it a bit and keep it only for the AWS marketplace?

Reply:
I removed the helm parts.